<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExampleWindow</class>
 <widget class="QMainWindow" name="ExampleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExampleWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="myButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="myTextBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>104</width>
      <height>74</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="myLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="myTable">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
